data/yaml/venues.yaml

```diff
@@ -780,6 +780,9 @@ rdsm:
 readi:
   acronym: READI
   name: Tools and Resources to Empower People with REAding Difficulties
+reinact:
+  acronym: ReInAct
+  name: Reasoning and Interaction Conference
 repeval:
```

Comment:
Is there a website for this conference? If so, we should add it.

Reply:
Yes, added